Resources/Prototypes/GameRules/roundstart.yml

```
@@ -269,6 +269,29 @@
       mindRoles:
       - MindRoleInitialInfected
 
+- type: entity
+  parent: Thief
+  id: OopsAllThieves
+  components:
+  - type: GameRule
+    minPlayers: 30
+  - type: AntagSelection
+    agentName: thief-round-end-agent-name
+    definitions:
+    - prefRoles: [ Thief ]
+      max: 10
+      playerRatio: 10
```

this means 10 players per 1 thief, on a wizden server you'd never reach 10 since we cap at 80 players.
```suggestion
      playerRatio: 1
```
Added a suggestion to make it *all* thieves so its more like a scavenger hunt gamemode.